aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -15,106 +15,106 @@ uid: host-and-deploy/windows-service
 
 ## Getting started
 
 This section explains the minimum changes required to set up an existing ASP.NET Core project to run in a service.
 
-* Install the NuGet package [Microsoft.AspNetCore.Hosting.WindowsServices](https://www.nuget.org/packages/Microsoft.AspNetCore.Hosting.WindowsServices/).
+1. Install the NuGet package [Microsoft.AspNetCore.Hosting.WindowsServices](https://www.nuget.org/packages/Microsoft.AspNetCore.Hosting.WindowsServices/).
 
-* Make the following changes in `Program.Main`:
+1. Make the following changes in `Program.Main`:
   
-  * Call `host.RunAsService` instead of `host.Run`.
+   * Call `host.RunAsService` instead of `host.Run`.
   
-  * If the code calls `UseContentRoot`, use a path to the publish location instead of `Directory.GetCurrentDirectory()` 
-  
-  [!code-csharp[](windows-service/sample/Program.cs?name=ServiceOnly&highlight=3-4,8,14)]
+   * If the code calls `UseContentRoot`, use a path to the publish location instead of `Directory.GetCurrentDirectory()`.
 
-* Publish the application to a folder.
+   # [ASP.NET Core 2.x](#tab/aspnetcore2x)
 
-  Use [dotnet publish](https://docs.microsoft.com/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles) that publishes to a folder.
+   [!code-csharp[](windows-service/sample/Program.cs?name=ServiceOnly&highlight=3-4,7,12)]
 
-* Test by creating and starting the service.
+   # [ASP.NET Core 1.x](#tab/aspnetcore1x)
 
-  Open an administrator command prompt window to use the [sc.exe](https://technet.microsoft.com/library/bb490995) command-line tool to create and start a service.  
-  
-  If the service is named MyService, publish the app to `c:\svc`, and the app itself is named AspNetCoreService, the commands would look like this:
+   [!code-csharp[](windows-service/sample_snapshot/Program.cs?name=ServiceOnly&highlight=3-4,8,14)]
+
+   ---
+
+1. Publish the app to a folder. Use [dotnet publish](/dotnet/articles/core/tools/dotnet-publish) or a [Visual Studio publish profile](xref:host-and-deploy/visual-studio-publish-profiles) that publishes to a folder.
 
-  ```console
-  sc create MyService binPath="C:\Svc\AspNetCoreService.exe"
-  sc start MyService
-  ```
+1. Test by creating and starting the service.
 
-  The `binPath` value is the path to the app's executable, including the executable filename itself.
+   Open an administrator command prompt window to use the [sc.exe](https://technet.microsoft.com/library/bb490995) command-line tool to create and start a service. If the service is named MyService, published to `c:\svc`, and named AspNetCoreService, the commands are:
```

Comment:
I suggest we use "command shell" instead of "command prompt window". How about "Open a command shell with administrative privileges..."?